uwb/example/lib/screens/game_screen.dart

```diff
@@ -0,0 +1,382 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:uwb/uwb.dart';
+
+import '../game_widgets/ball.dart';
+import '../game_widgets/brick.dart';
+import '../game_widgets/brick_field.dart';
+import '../game_widgets/paddle.dart';
+import '../game_widgets/score.dart';
+import '../game_widgets/screen_overlay.dart';
+
+/// The [BreakoutGame] class.
+///
+/// Contains the code for the breakout game.
+class BreakoutGame extends StatefulWidget {
+  /// The [BreakoutGame] constructor.
+  const BreakoutGame({Key? key}) : super(key: key);
+
+  @override
+  BreakoutGameState createState() => BreakoutGameState();
+}
+
+/// Contains the direction enums of the ball.
+enum Direction {
+  /// Ball goes up.
+  up,
+
+  /// Ball goes down.
+  down,
+
+  /// Ball goes left.
+  left,
+
+  /// Ball goes right.
+  right
+}
+
+/// The [BreakoutGameState] class.
+class BreakoutGameState extends State<BreakoutGame> {
```

Comment:
In almost all cases the state should be private to the library and not be exposed to the public. Consider renaming this to:

```suggestion
class _BreakoutGameState extends State<BreakoutGame> {
```